requirements.txt

```diff
@@ -1,32 +1,4 @@
-certifi==2020.12.5
-cffi==1.14.4
-chardet==3.0.4
-click==7.1.2
-cryptography==3.3.1
 Flask==1.1.2
-Flask-Login==0.5.0
-gcloud==0.17.0
-googleapis-common-protos==1.52.0
-httplib2==0.18.1
-idna==2.10
-itsdangerous==1.1.0
-Jinja2==2.11.2
-jws==0.1.3
-MarkupSafe==1.1.1
-oauth2client==3.0.0
 passlib==1.7.4
-protobuf==3.14.0
-pyasn1==0.4.8
-pyasn1-modules==0.2.8
-pycparser==2.20
-pycryptodome==3.4.3
 PyJWT==1.7.1
-Pyrebase==3.0.27
-python-dotenv==0.15.0
```

Comment:
This module is needed. It reads the key,value pair from .env and adds them to environment variable. 
Link to the [docs](https://pypi.org/project/python-dotenv/)

Reply:
I knew from the beginning that there was something wrong with the file __sample.env__. I have added the library again but there is still the question where in this repository the library is used. Could you advise me with further instructions please?